# happens when „1‟== 1 is executed? 

#Easy explanation - It simply evaluates to False and does not raise any exception